documentation/modules/deploying/proc-deploy-setup-external-clients.adoc

```diff
@@ -203,11 +203,11 @@ metadata:
     strimzi.io/cluster: my-cluster
 type: Opaque
 data:
-  ca.crt: _PUBLIC-KEY-OF-THE-CLIENT-CA_
-  user.crt: _USER-CERTIFICATE-CONTAINING-PUBLIC-KEY-OF-USER_
-  user.key: _PRIVATE-KEY-OF-USER_
-  user.p12: _P12-ARCHIVE-FILE-STORING-CERTIFICATES-AND-KEYS_
-  user.password: _PASSWORD-PROTECTING-P12-ARCHIVE_
+  ca.crt: # Public key of the client CA
+  user.crt: # User certificate that contains the public key of the user
+  user.key: # Private key of the user
+  user.p12: # PKCS #12 archive file for storing certificates and keys
+  user.password: # Password for protecting the PKCS #12 archive file
```

Comment:
Does the YAML formatting still work for this? SInce I don't think this is a valid YAML anymore. MAybe you can use some placeholders?

```suggestion
  ca.crt: <...> # Public key of the client CA
  user.crt: <...> # User certificate that contains the public key of the user
  user.key: <...> # Private key of the user
  user.p12: <...> # PKCS #12 archive file for storing certificates and keys
  user.password: <...> # Password for protecting the PKCS #12 archive file
```

Reply:
Changed to this in the 3 places the example is used:

> ca.crt: _<public_key>_ # Public key of the clients CA
>   user.crt: _<user_certificate>_ # Public key of the user
>   user.key: _<user_private_key>_ # Private key of the user
>   user.p12: _<store>_ # PKCS #12 store for user certificates and keys
>   user.password: _<password_for_store>_ # Protects the PKCS #12 store